packages/libs/coreui/src/components/Mesa/Ui/DataTable.jsx

```diff
@@ -123,11 +122,9 @@ class DataTable extends React.Component {
     if (this.props.rows.length === 0 || this.props.filteredRows.length === 0)
       return;
 
-    this.setState({ dynamicWidths: null, tableWrapperWidth: null }, () => {
-      this.widthCache = {};
-      const { columns } = this.props;
-      const hasSelectionColumn = this.hasSelectionColumn();
-      const { contentTable, getInnerCellWidth } = this;
+    this.setState({ dynamicWidths: null }, () => {
```

Comment:
I'm confused as to where `this.state.dynamicWidths` is ever accessed?

This seems to be more about setting `this.widthCache`.

Reply:
Thanks for catching. That was an oversight. I meant to purge all occurrences of dynamicWidths. Will fix in a follow-up.